src/Features/Core/Portable/RemoveUnusedParametersAndValues/AbstractRemoveUnusedValuesCodeFixProvider.cs

```diff
@@ -546,7 +555,23 @@ protected sealed override async Task FixAllAsync(Document document, ImmutableArr
 
                 foreach (var node in nodesToRemove)
                 {
-                    editor.RemoveNode(node, SyntaxGenerator.DefaultRemoveOptions | SyntaxRemoveOptions.KeepLeadingTrivia);
+                    var removeOptions = SyntaxGenerator.DefaultRemoveOptions;
```

Comment:
I struggled a bit getting the right combination for the remove options, glad you got this working!
I wonder if this also addresses the pending trivia related issues mentioned in https://github.com/dotnet/roslyn/issues/31398#issue-384960358. If so, you should close that issue as well. Thanks!

Reply:
Sure!  I'll go through the pending ones.  If any are fixed I'll update this PR with test cases that confirm the behavior.

Reply:
It looks like these are caused by a slightly different problem - the node isn't getting immediately removed, but an add -> replace -> remove combination.  It is similar to an issue I encountered before, so I'll see if it's easy to fix in this PR.

Reply:
In those other issues, it looks like the original new lines are getting replaced by elastic new lines somehow.  I've spent a few hours trying to figure out how they're getting replaced but haven't made much progress.

For now I'll merge at least this fix and look at the others if I have time later (or I find some inspiration).

Reply:
Sounds good to me. Thanks!